targets/TARGET_STM/rtc_api.c

```diff
@@ -123,6 +124,10 @@ void rtc_init(void)
     RtcHandle.Init.OutPutType     = RTC_OUTPUT_TYPE_OPENDRAIN;
 #endif /* TARGET_STM32F1 */
 
+#if DEVICE_LOWPOWERTIMER && !MBED_CONF_TARGET_LOWPOWERTIMER_LPTIM
+    lp_TickPeriod_us = 2048 * 1000000 / RtcHandle.Init.SynchPrediv;
```

Comment:
where is 2048 coming from ? would it make sense to have a macro here with name that would explain ?

Reply:
Done, thx